code/game/gamemodes/changeling/powers/tiny_prick.dm

```diff
@@ -246,3 +246,21 @@
 		target.reagents.add_reagent("frostoil", 30)
 	feedback_add_details("changeling_powers","CS")
 	return 1
+
+
+/obj/effect/proc_holder/changeling/sting/para
+	name = "Paralysis Sting"
+	desc = "We silently sting a human to paralyse them."
+	helptext = "Will weaken a target for 20 ticks."
+	sting_icon = "sting_cryo"
+	chemical_cost = 30
+	dna_cost = 2
+
+/obj/effect/proc_holder/changeling/sting/para/sting_action(mob/user, mob/target)
+	add_logs(user, target, "stung", "para sting")
+	if(istype(target, /mob/living/carbon))

```

Comment:
You shouldn't need to istype the target, as line 37 already checks if the target is a carbon mob.